docs/book/v3/features/template/laminas-view.md

```diff
@@ -18,164 +18,288 @@ To use the laminas-view wrapper, you must first install the laminas-view integra
 $ composer require mezzio/mezzio-laminasviewrenderer
 ```
 
+## Available Versions
+
+The following table shows the compatibility of the libraries that go together to provide a usable templating system using `laminas-view`:
+
+| `laminas-view` | `mezzio-template` | `mezzio-laminasviewrenderer` |
+|----------------|-------------------|------------------------------|
+| 2.x            | 2.x               | 2.x                          |
+| 3.x            | 3.x               | 3.x                          |
+
+There are significant changes in `laminas-view` version 3, but broadly speaking, its usage in a Mezzio application is largely unchanged.
+
+This documentation will cover usage of `laminas-view` with the version 3 series of releases.
```

Comment:
A link can be added to version 2, as both files were identical: https://docs.mezzio.dev/mezzio/v2/features/template/laminas-view/